packages/beacon-node/src/metrics/validatorMonitor.ts

```diff
@@ -287,6 +289,53 @@ export function createValidatorMonitor(
       }
     },
 
+    registerValidatorStatuses(currentEpoch, statuses) {
+      // Prevent registering status for the same epoch twice. processEpoch() may be ran more than once for the same epoch.
+      if (currentEpoch <= lastRegisteredStatusEpoch) {
+        return;
+      }
+      lastRegisteredStatusEpoch = currentEpoch;
+
+      const statusCount = {
```

Comment:
For these metrics, I'd recommend just using the raw statuses and not using these aggregated ststuses.

Reply:
The intention was to [mimic lighthouse's metrics](https://github.com/ChainSafe/lodestar/issues/5158#issuecomment-1452597916) given consumers may be already acquainted with this format.

Reply:
I think the logic for selecting statuses needs to be updated.

There are some raw statuses that translate to multiple aggregated statuses.

Eg:
active_slashed = active and slashed
exited_slashed = exited and slashed
etc